intranet/templates/eighth/take_attendance.html

```diff
@@ -51,6 +51,21 @@
     });
     </script>
     {% endif %}
+    <script>
+        function openBbcuTab() {
+            var student_list = [];
+            var mem_list = document.getElementsByClassName("bbcu-selector");
```

Comment:
You can use jQuery to replace some of these native JS functions

Reply:
Haven't written much in jQuery, is the updated syntax for it about right?  (I did test it and it works)